--- index.js	2020-08-19 21:55:59.206745243 +0200
+++ ../ssb-browser-core/node_modules/ssb-blob-files/index.js	2020-08-19 21:45:09.697773451 +0200
@@ -33,7 +33,7 @@
 
   function buildFileDoc (file, cb) {
     if (resize || file.size < maxSize) {
-      const reader = new global.FileReader()
+      const reader = new FileReader()
       reader.onload = function (e) {
         cb(null, {
           name: file.name,
