--- node_modules/ssb-ebt/index.js~	1985-10-26 09:15:00.000000000 +0100
+++ node_modules/ssb-ebt/index.js	2020-06-01 23:43:33.262830200 +0200
@@ -85,28 +85,6 @@
     ebt.onAppend(msg.value)
   })
 
-  //HACK: patch calls to replicate.request into ebt, too.
-  hook(sbot.replicate.request, function (fn, args) {
-    var id, replicate
-    if(isObject(args[0])) {
-      id = args[0].id
-      replicate = args[0].replicate
-    }
-    else {
-      id = args[0]
-      replicate = args[1]
-    }
-    if(!isFeed(id)) return
-    ebt.request(id, replicate)
-    return fn.apply(this, args)
-  })
-
-//  hook(sbot.status, function (fn) {
-//    var _status = fn(), feeds = 0
-//    _status.ebt = ebt.status()
-//    return _status
-//  })
-
   hook(sbot.progress, function (fn) {
     var prog = fn()
     var p = ebt.progress()
@@ -152,14 +130,6 @@
       sbot.gossip && sbot.gossip.disconnect(to, function () {})
   }
 
-  if(sbot.replicate.block)
-    sbot.replicate.block.hook(function (fn, args) {
-      block.apply(this, args)
-      return fn.apply(this, args)
-    })
-  else
-    Legacy(sbot, ebt) //remove this on next breaking change
-
   return {
     replicate: function (opts) {
       if(opts.version !== 2 && opts.version != 3)
@@ -187,6 +157,8 @@
       return data
     },
 
+    request: ebt.request,
+
     // expose ebt.block for ssb-servers that don't have the ssb-friends plugin
     block: block
   }
